### DISCUSSION TITLE
posts/2022-07-08-pinky

### DISCUSSION THREAD

**Discussion by giscus** [cmdcolin/cmdcolin.github.io#6] [General]
# posts/2022-07-08-pinky

Blogging for the future

https://cmdcolin.github.io/posts/2022-07-08-pinky#comments

**Comment by mtsknn** [comment]
Have you tried remapping Alt to Ctrl? You would get Mac-style shortcuts, e.g. Alt+X/C/V for cutting/copying/pasting. Pressing Alt with the thumb might feel more comfortable than pressing Ctrl with the pinky finger.

You could try e.g. https://github.com/rbreaves/kinto ("Mac-style shortcut keys for Linux & Windows"), though I haven't personally used it as I use Mac.

**Reply by cmdcolin** [reply]
thanks for the idea :) I'm also glad this works for commenting, you're the first person to comment on a post. at this stage I'd probably not use that as i'm sort of interested in making right hand shortcuts, or using both hands (a la http://www.rsi.deas.harvard.edu/preventing.html). even things that seem pretty "easy" for the left hand can cause issues pretty quickly. I'm taking a longer vacation soon also which will hopefully help too. but, in general, got to also work on my posture and getting a better desk setup. I type on a laptop keyboard (Dell PC) that is not particularly ergonomic and also has me using all matter of surfaces as desks which I enjoy, but it probably contributes to my issues
